src/dialogs/more-info/controls/more-info-fan.ts

```diff
@@ -87,6 +87,9 @@ class MoreInfoFan extends LitElement {
 
   _handleOscillating(ev) {
     const newVal = ev.target.value === "true";
+    const oldVal = this.stateObj?.attributes.oscillating;
+
+    if (!newVal || oldVal === newVal) return;
```

Comment:
This means oscillation can never be turned off?

Reply:
oh sorry, I made a mistake, I just copy the code from _handlePresetMode.